cms/migrations/0017_create_and_setup_home_page.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 2.2.10 on 2020-06-09 10:12
+
+from django.db import migrations
+from django.contrib.contenttypes.models import ContentType
+from wagtail.core.models import Page, Site
+
+from cms.models import HomePage
+
+
+DEFAULT_WAGTAIL_HOMEPAGE_PROPS = dict(
+    title="Welcome to your new Wagtail site!", depth=2, path="00010001"
+)
+DEFAULT_HOMEPAGE_PROPS = dict(
+    title="MIT Bootcamps", tagline="Learn Innovation and Entrepreneurship from MIT"
+)
+DEFAULT_SITE_PROPS = dict(hostname="localhost", port=80)
+
+
+def create_home_page(apps, schema_editor):
+    """
+    Create the home page
+    """
+    site = Site.objects.filter(is_default_site=True).first()
```

Comment:
This can be optimized a bit

```python
from django.conf import settings

def create_home_page(apps, schema_editor):
    # ... get model classes from 'apps' ...
    root = Page.objects.get(depth=1)
    valid_home_page = Page.objects.filter(
        content_type=ContentType.objects.get_for_model(HomePage)
    ).first()
    if valid_home_page is None:
        valid_home_page = HomePage(**DEFAULT_HOMEPAGE_PROPS)
        root.add_child(instance=valid_home_page)
        valid_home_page.refresh_from_db()
    site, _ = Site.objects.get_or_create(
        is_default_site=True,
        defaults=dict(
            root_page=valid_home_page,
            site_name=settings.WAGTAIL_SITE_NAME,
            **DEFAULT_SITE_PROPS
        )
    )
    if site.root_page != valid_home_page:
        site.root_page = valid_home_page
        site.save()
    wagtail_default_home_page = Page.objects.filter(
        depth=2, content_type=ContentType.objects.get_for_model(Page)
    ).first()
    # ...
```